### DISCUSSION TITLE
Isolation not working

### DISCUSSION THREAD

**Discussion by kanonkanin** [Azure/Bridge-To-Kubernetes#300] [Q&A] [answered]
Hi.

I have tried Bridge to Kubernetes, and it works well in normal operation. But, when I try Isolation, nothing happens.
I have tried with "header" propagation in a project where http client is added, and also on a simpler application where there´s no http client added.

It seems to start ok, (some output below), but it is not redirected as without. Are there any other configuration that I should be aware of?

Redirecting Kubernetes service certification-service to your machine... 
Target cluster: xx
Current cluster: xx
Target namespace: test
Current namespace: test
Target service name: x-service
Target service ports: 80
Isolating service with routing header: xx-5992
Using kubernetes service environment variables: false

**Comment by hsubramanianaks** [comment, marked as answer]
@kanonkanin Can you please raise an issue with logs from ```/tmp/Bridge To Kubernetes``` folder ?

**Comment by kanonkanin** [comment]
Never mind, I figured it out, I was unclear how to set it up in Azure with regard to DNS and so on.
